internal/providers/terraform/azure/testdata/redis_cache_test/redis_cache_test.golden

```diff
@@ -0,0 +1,16 @@
+
+ Name                                  Monthly Qty  Unit   Monthly Cost 
+                                                                        
+ azurerm_redis_cache.basic_c2                                           
+ └─ Cache usage (Basic_C4), 4 node           2,920  hours       $613.20 
```

Comment:
`(Basic_C4, 4 nodes)` would be more consistent with other cost component names. Also can we plz add a tiny bit of logic so it shows `1 node` and `2 nodes` etc